client-src/src/pages/counselor/inq-list/inq-list.html

```diff
@@ -38,11 +38,67 @@
     </ion-row>
   </ion-toolbar>
 
-  <ion-card *ngFor = "let inq of inquiries; let i = index;">
+  <ion-card color="light">
+    <ion-row>
+      <ion-col col-1 class="vertical-center-contents">
+        <div class="header-title" (click)="toggleSort('id')">
+          Inquiry ID
+          <span class="sort-icon" *ngIf="sortBy.id.bool">
+              <ion-icon *ngIf = "sortBy.id.ascending" name="ios-arrow-down" class="see-more"></ion-icon>
+              <ion-icon *ngIf = "sortBy.id.descending" name="ios-arrow-up" class="see-more"></ion-icon>
+          </span>
+        </div>
+      </ion-col>
+      <ion-col col-1 class="vertical-center-contents">
+        <div class="header-title" (click)="toggleSort('caseIndex')">
+          Case index
+          <span class="sort-icon" *ngIf="sortBy.caseIndex.bool">
```

Comment:
This code is common for all. Avoid duplicating. May use directive or a common method to toggle

Reply:
the toggleSort() method?

Reply:
This code:

<span class="sort-icon" *ngIf="sortBy.id.bool">
 +              <ion-icon *ngIf = "sortBy.id.ascending" name="ios-arrow-down" class="see-more"></ion-icon>
 +              <ion-icon *ngIf = "sortBy.id.descending" name="ios-arrow-up" class="see-more"></ion-icon>
 +          </span>

Reply:
<span class="sort-icon" *ngIf="sortBy.id.bool">
<ion-icon *ngIf = "sortBy.id.ascending" name="ios-arrow-down" class="see-more"></ion-icon>
<ion-icon *ngIf = "sortBy.id.descending" name="ios-arrow-up" class="see-more"></ion-icon>
</span>